test/proper_tests.erl

```diff
@@ -40,8 +40,8 @@
 
 %% NOTE: Never add long_result to Opts for these macros.
 
-state_is_clean() ->
-    get() =:= [].
+-define(state_is_clean(), ?assertEqual([],get())).
+-define(_state_is_clean(), ?_assertEqual([],get())).
```

Comment:
what is the advantage of using state_is_clean() to using ?assert(state_is_clean())? For me the old notation is easier to read (an assertion that checks that the state is clean)

Reply:
I initially prefixed it with `assert_` but then thought it was getting too long. Would you be happy if I prefixed it?

The advantage is to show what’s in the get() when it’s not empty and to report the line where the macro is called.